Mentorship Guidelines.md

```diff
@@ -0,0 +1,154 @@
+<img src="https://user-images.githubusercontent.com/82679007/150870419-21f7d802-0e40-4a0c-b16a-17f37a18115c.png" alt="The Galaxy Mentor Network logo." width="400"  align="right" />
+
+# Galaxy Mentorship Network Guidelines
+
+Welcome to the Galaxy Community Mentorship Program designed to help community members who may be bioinformaticians, data analysts, students, programmers, e.t.c, become more proficient in the use of Galaxy tools and integration into the Galaxy Community.
+
+## Program Duration
+
+A typical mentorship program is scheduled to run for 8 weeks. Within this 8-week period, GMentors will work closely with GMentees and interact with other GMentors in the mentor community chat.
+
+Highlights of this mentorship program include:
+
+- Week 1 - an introductory call between GMentees, GMentors, and Network Managers.
+- Week 4 - mid-program feedback
+- Week 8 - final feedback.
+
+## Program Tracking
+
+**How we track progress**
```

Comment:
Should it be a subsection?

Reply:
Yes, it is...the same font as the paragraph but just bolder to show hierarchy. I'll appreciate tips on a more appropriate way to present it. Thank you:)

Reply:
You could make it like proper subsection with `###` instead of the bold (same for other subsections after)

Reply:
Then it would make the possibility to link to them